boards/arm/cc3235s_launchxl/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# TI SimpleLink CC3235S LaunchXL Board
+
+# Copyright (c) 2019, Texas Instruments Incorporated
```

Comment:
@ithinx-io - are you an employee of TI? If not, I would suggest adding your own name to any copyrights.

Reply:
Ok, no, I am not. I will fix it.